modules/tinymce/src/core/main/ts/html/Base64Uris.ts

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright (c) Tiny Technologies, Inc. All rights reserved.
+ * Licensed under the LGPL or a commercial license.
+ * For LGPL see License.txt in the project root for license information.
+ * For commercial licenses see https://www.tiny.cloud/
+ */
+
+import { Id, Obj } from '@ephox/katamari';
+
+export interface Base64UriMatch {
+  mime: string;
+  base64: string;
+}
+
+export interface Base64Extract {
+  prefix: string;
+  uris: Record<string, Base64UriMatch>;
+  html: string;
+}
+
+export const extractBase64DataUris = (html: string): Base64Extract => {
```

Comment:
Another thing I just noticed, this name is very misleading, as it doesn't extract base64 data URIs and instead only extracts **image** base64 URIs.

- [ ] Consider renaming this, or make it work with all mime types, not just images.